libchannelserver/ChannelSession.cpp

```diff
@@ -122,7 +122,7 @@ void ChannelSession::asyncSendMessage(Message::Ptr request,
 
                 responseCallback->timeoutHandler = timeoutHandler;
             }
-            _responseCallbacks.insert(std::make_pair(request->seq(), responseCallback));
+            insertResponseCallback(request->seq(), responseCallback);
```

Comment:
Only use once insert, is it necessary to add a new function?

Reply:
Add a new function will increase the extension of the code, but decrease the 
readability。